src/snowflake/cli/plugins/nativeapp/codegen/snowpark/python_processor.py

```diff
@@ -510,7 +510,9 @@ def edit_setup_script_with_exec_imm_sql(
             sql_file_relative_path = sql_file.relative_to(
                 deploy_root
             )  # Path on stage, without the leading slash
-            file.write(f"EXECUTE IMMEDIATE FROM '/{sql_file_relative_path}';\n")
+            file.write(
```

Comment:
line 231 stays as is, with output like:
`Generated by the Snowflake CLI from cli_gen\accepted.py`
Do we also want to change this comment to be stage path?

Reply:
"generated from" should be the source file, i.e. a local path in the project root, not a dest path / stage path.